lambda-on-binaris/binaris.d.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="node" />
```

Comment:
why is this here?
isn't this a duplicate of https://github.com/binaris/binaris/blob/master/lib/functionTemplates/js/binaris.d.ts ?

oh, it's part of the new template. didn't occur to me this can daunting for javascript people. we should instead from the binaris package instead of importing type locally from a file with `@type { import("./binaris").BinarisFunction }`. this is not nice.